docs-src/cloud-context/saml-intro.md

```diff
@@ -0,0 +1,24 @@
+---
+id: saml-intro
+title: How to manage SAML authentication with Temporal Cloud
+sidebar_label: Manage SAML
+description: To authenticate users of your Temporal Cloud account, you can connect a SAML IdP.
+tags:
+  - guide-context
+---
+
+To authenticate the users of your Temporal Cloud account, you can connect a Security Assertion Markup Language (SAML) identity provider (IdP).
+
+If you want to use the general Microsoft login mechanism, you don't need to set up SAML with Azure AD.
+Just select **Continue with Microsoft** on the Temporal Cloud sign-in page.
+
+## Integrate SAML with your Temporal Cloud account
+
+1. Locate your [Temporal Cloud Account Id](/concepts/what-is-a-cloud-account-id).
+   One way to do so is to sign in to Temporal Cloud and find your [Namespace Id](/concepts/what-is-a-cloud-namespace-id).
```

Comment:
There is an edge case where the account doesn't have a namespace yet--so they cannot use this method to find their account id. We should sync with product/design on a way for users to get the id otherwise.